src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball [] setOfBalls = new Ball [3];
```

Comment:
```suggestion
        Ball[] setOfBalls = new Ball[3];
```
autoformat this file

try to rewrite this class without using the array
create only one lottery instance (outside of this loop), right now you create three of them
print result of the calling Lottery's method directly inside of a loop